tests/topotests/grpc_basic/test_basic_grpc.py

```diff
@@ -149,6 +149,13 @@ def test_get_config(tgen):
         }
       }
     ]
+  },
```

Comment:
again, maybe the test should be more future-looking, and not demand an exact comparison for every attribute of every object.